# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363190890 25200
#      Wed Mar 13 09:08:10 2013 -0700
# Node ID ef39f8b1c5c83bf5afb8bbda2a8a0e4c82fe78e2
# Parent  4a1300ed5d3cd07b87f38529256a36c699aa02f5
configure.ac: Unconditionally define AMCOND_HAVE_QSCINTILLA.

* configure.ac: Move macro definition for AMCOND_HAVE_QSCINTILLA out
of conditional block so that it is always defined.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2618,18 +2618,16 @@ if test $build_gui = yes; then
     else
       ## Let's assume Qscintilla library is at the same location as
       ## other regular Qt libraries.
       QT_LIBS="$QT_LIBS -lqscintilla2"
       OCTAVE_CHECK_FUNC_FINDFIRST_MODERN
       AC_DEFINE(HAVE_QSCINTILLA, 1, 
         [Define to 1 if the QScintilla library and header files are available])
     fi
-    AM_CONDITIONAL([AMCOND_HAVE_QSCINTILLA],
-      [test $octave_cv_lib_qscintilla = yes])
 
     AC_CHECK_FUNCS([setlocale], [],
       [AC_MSG_WARN([setlocale not found -- disabling GUI])
        build_gui=no])
 
     if test $build_gui = yes; then
       case $host_os in
         mingw* | msdosmsvc*)
@@ -2650,16 +2648,17 @@ if test $build_gui = yes; then
   fi
 fi
 
 if test $build_gui = no; then
   ## GUI disabled.  Eliminate building GUIDIR directory
   GUIDIR=
 fi
 AM_CONDITIONAL([AMCOND_BUILD_GUI], [test $build_gui = yes])
+AM_CONDITIONAL([AMCOND_HAVE_QSCINTILLA], [test $octave_cv_lib_qscintilla = yes])
 AM_CONDITIONAL([WIN32_TERMINAL], [test $win32_terminal = yes])
 AC_SUBST(QT_CPPFLAGS)
 AC_SUBST(QT_LDFLAGS)
 AC_SUBST(QT_LIBS)
 AC_SUBST(GUIDIR)
 
 ### Run configure in subdirectories.
 
